appsDiscovery/src/main/java/org/kinecosystem/appsdiscovery/view/AppInfoActivity.kt

```diff
@@ -24,15 +24,15 @@ import org.kinecosystem.appsdiscovery.view.customView.AppImagesListAdapter
 import org.kinecosystem.appsdiscovery.view.customView.AppStateView
 import org.kinecosystem.appsdiscovery.view.customView.TransferBarView
 import org.kinecosystem.appsdiscovery.view.customView.TransferInfo
-import org.kinecosystem.appsdiscovery.model.*
-import org.kinecosystem.appsdiscovery.repositories.DiscoveryAppsLocal
-import org.kinecosystem.appsdiscovery.repositories.DiscoveryAppsRemote
-import org.kinecosystem.appsdiscovery.repositories.DiscoveryAppsRepository
+import org.kinecosystem.transfer.repositories.DiscoveryAppsLocal
+import org.kinecosystem.transfer.repositories.DiscoveryAppsRemote
+import org.kinecosystem.transfer.repositories.DiscoveryAppsRepository
```

Comment:
How about we rename these three to:
`EcosystemAppLocalStore` , `EcosystemAppRemoteStore`, `EcosystemAppRepository`
(note that an *app* store can contain many apps, no need to add the `s`)

Reply:
cool